src/server/db_slice.cc

```diff
@@ -326,7 +332,7 @@ void DbSlice::Reserve(DbIndex db_ind, size_t key_size) {
 
 auto DbSlice::Find(const Context& cntx, string_view key, unsigned req_obj_type) const
     -> OpResult<PrimeIterator> {
-  auto it = FindExt(cntx, key).first;
+  auto it = FindInternal(cntx, key, FindInternalMode::kUpdateStats).first;
```

Comment:
I believe this also should be kDontUpdateStats right?

Reply:
`Find()` is about to be removed (https://github.com/dragonflydb/dragonfly/pull/2290), so it doesn't matter greatly, but sure :)